lib/geordi/util.rb

```diff
@@ -62,21 +62,28 @@ def system!(*commands)
         end
       end
 
+      def binstub(executable, *arguments)
+        binstub_file = "bin/#{executable}"
+
+        command = File.exists?(binstub_file) ? [binstub_file] : ['bundle exec', executable]
+        command.append(*arguments).join(' ')
```

Comment:
Again, I believe `append` does not work in older Rubies. Please use `push` instead:

```suggestion
        command.push(*arguments)
        command.join(' ')
```